### PR TITLE
Authorization audit

### PR DESCRIPTION
**Pull request by Senjai**

### CODE REVIEW

File: app/overrides/decorate_admin_configurations_index.rb [hunk 1]
```diff
@@ -1,5 +1,11 @@
...
+Deface::Override.new(
+  virtual_path: "spree/admin/shared/_configuration_menu",
+  name: "multi_domain_admin_configurations_menu",
+  insert_bottom: "[data-hook='admin_configurations_sidebar_menu']",
+  disabled: false,
+  text: <<-EOF
+  <% if can?(:display, Spree::Store) %>
+    <%= configurations_sidebar_menu_item Spree.t(:stores_admin), admin_stores_url %>
+  <% end %>
+  EOF
```
Comment (gmacdougall) [inline app/overrides/decorate_admin_configurations_index.rb]: Can you extract this to a template?

It makes it easier to override if you want to change this.

**Comment by athal7** [comment]
:+1:

File: app/overrides/decorate_admin_configurations_index.rb (hunk 1 above)
Comment (Senjai) [inline app/overrides/decorate_admin_configurations_index.rb]: Sure thing.

**Comment by cbrunsdon** [comment]
:+1: 

Will be interesting to start tackling admin duties for specific stores some time in the future.

**Pull request merged**